tools/schemas/app-config.schema.json

```diff
@@ -0,0 +1,98 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "SGX-LKL app config schema",
+    "required": [
+        "run",
+        "args",
```

Comment:
With a default value, why is `args` required?

Reply:
Good point. I'll remove it.